engine-spring/core/src/test/java/org/camunda/bpm/engine/spring/test/SpringProcessEngineInitializationTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.spring.test;
```

Comment:
I think we can move this test case to a sub-package like `configuration` or `context`, what do you think?

Reply:
I think we can move `testSpringConfigurationContextInitializing` to `configuration` package (also maybe we need rename `testSpringConfigurationContextInitializing` to `shouldBuildSpringContextConfiguration`?) and `shouldInitializeProcessEngineFromActivitiContext` we can move to `initializing` package.

Reply:
I'd say we can move this into the `configuration` package that already exists.